crux_core/src/bridge/request_serde.rs

```diff
@@ -87,7 +87,8 @@ impl<Out> Resolve<Out> {
             Resolve::Never => ResolveSerialized::Never,
             Resolve::Once(resolve) => ResolveSerialized::Once(Box::new(move |deser| {
                 let out = func(deser)?;
-                Ok(resolve(out))
+                resolve(out);
+                Ok(())
```

Comment:
Do we need to align the error handling across the `Once` and `Many`? It's quite suspicious that one returns its `resolve` return and the other doesn't... 🤔 

I vaguely remember needing to mess with this for the error handling changes recently, but now I can't remember the details. But I am a little concerned about changing this - can't immediately see what the consequences are.

Reply:
That resolve returns the unit type, it's only `ResolveMany` that returns a result (to carry an error from trying to resolve when already finished). So nothing changes. Just fixes a clippy warning.

Reply:
Alright. I guess we need to revisit whether it _should_ return a result, but not in this PR.